plugins/modules/foreman_location.py

```diff
@@ -107,14 +107,14 @@ def main():
     module = ForemanLocationModule(
         foreman_spec=dict(
             name=dict(required=True),
-            parent=dict(type='entity'),
+            parent=dict(type='entity', resource_type='locations'),
```

Comment:
can't `parent` be guessed from `entity` as it's the same resource type?

Reply:
As we do some guesswork around parent anyway, why not.